internal/pkg/platform/activity_test.go

```diff
@@ -147,37 +148,37 @@ func TestPlatformActivity_StreamingLogs(t *testing.T) {
 				}()
 				return ctx
 			},
-			ExpectedAsserts: func(t *testing.T, cm *shared.ClientsMock) {
+			ExpectedAsserts: func(t *testing.T, ctx context.Context, cm *shared.ClientsMock) {
 				// with the above polling/canceling setup, expectation is activity called only once.
 				cm.ApiInterface.AssertNumberOfCalls(t, "Activity", 1)
 			},
 		},
 	} {
 		t.Run(name, func(t *testing.T) {
-			ctx := context.Background()
-			ctx = context.WithValue(ctx, config.CONTEXT_TOKEN, "sometoken")
 			// Create mocks
+			ctxMock := slackcontext.MockContext(context.Background())
+			ctxMock = context.WithValue(ctxMock, config.CONTEXT_TOKEN, "sometoken")
 			clientsMock := shared.NewClientsMock()
 			// Create clients that is mocked for testing
 			clients := shared.NewClientFactory(clientsMock.MockClientFactory())
 			// Setup custom per-test mocks (higher priority than default mocks)
 			if tt.Setup != nil {
-				ctx = tt.Setup(t, ctx, clientsMock)
+				ctxMock = tt.Setup(t, ctxMock, clientsMock)
```

Comment:
thought: I noticed that this activity test appears to have copy & pasted the `TableTestCommand` implementation, then modified it to work for an `internal/pkg/` test. Probably something to revisit in the future because it could be simplified to match our other non-command table tests.

Reply:
🎁 Super intersting find @mwbrooks! I agree we should revisit these tests at some point since it's also surprising to find `context` changes here.

Untangling that might mean leaning more into passing `app` and `auth` as arguments but I don't recall the underhappenings of this package right now! It's no blocker to the changes otherwise either!